x/faucet/ante.go

```diff
@@ -0,0 +1,23 @@
+package faucet
+
+import (
+	"github.com/Sifchain/sifnode/x/clp"
+	types2 "github.com/Sifchain/sifnode/x/faucet/types"
+	"github.com/cosmos/cosmos-sdk/types"
+)
+
+type RemoveFaucetFeeDecorator struct{}
+
+func NewRemoveFacuetFeeDecorator() RemoveFaucetFeeDecorator {
+	return RemoveFaucetFeeDecorator{}
+}
+
+var _ types.AnteDecorator = RemoveFaucetFeeDecorator{}
+
+func (r RemoveFaucetFeeDecorator) AnteHandle(ctx types.Context, tx types.Tx, simulate bool, next types.AnteHandler) (newCtx types.Context, err error) {
```

Comment:
I would add a small godoc here illustrating what is happening. e.g. `// On a testnet only where the chain-id is not equal to "sifchain" ...`